osu!stream/Support/Desktop/GameBaseDesktop.cs

```diff
@@ -59,14 +59,20 @@ protected override SoundEffectPlayer InitializeSoundEffects()
 
         protected override void InitializeInput()
         {
+
+            //InputManager.AddSource(new InputSourceMouse(Window.Mouse));
+            InputSource source;
+
             try
             {
-                InputManager.AddSource(new InputSourceBaanto());

```

Comment:
you've removed support for the baanto touch screen, which we still use. raw input should be a fallback.

Reply:
Oh, my bad.